src/core/server/base.js

```diff
@@ -13,19 +13,21 @@ import { match } from 'react-router';
 import { ReduxAsyncConnect, loadOnServer } from 'redux-async-connect';
 
 import WebpackIsomorphicTools from 'webpack-isomorphic-tools';
-import WebpackIsomorphicToolsConfig from 'config/webpack-isomorphic-tools';
+import WebpackIsomorphicToolsConfig from 'webpack-isomorphic-tools-config';
 
 import config from 'config';
 import { setJWT } from 'core/actions';
 
+const env = config.util.getEnv('NODE_ENV');
+const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
```

Comment:
We don't currently build the files in the same way when webpack is running the dev server. Therefore the json file that the webpack plugin builds doesn't exist which is why it's currently only doing SRI when we have built the files.

The webpack dev server serves all files from memory so I'm not sure if the sri plugin we're using can get the necessary data. If it can then fixing development to always run SRI should be doable, but not a huge priority. I'll file and issue and when we've got the higher priority stuff done we can look at it. In the meamtime if you want SRI there's nothing stopping anyone building and running the server in that way via the config and with the env var overrides as needed.